app/models/email_settings.rb

```diff
@@ -7,12 +7,12 @@ def get_values(conference, user, event = nil)
       'conference_start_date' => conference.start_date,
       'conference_end_date' => conference.end_date,
       'registrationlink' => Rails.application.routes.url_helpers.conference_conference_registrations_url(
-                            conference.short_title, host: CONFIG['url_for_emails']),
+                            conference.short_title, host: (ENV['OSEM_HOSTNAME'] || 'localhost:3000')),
```

Comment:
Actually I don't like to hardcode ``(ENV['OSEM_HOSTNAME'] || 'localhost:3000')``
(same for all the others)